R/estimatr_iv_robust.R

```diff
@@ -246,6 +246,8 @@ iv_robust <- function(formula,
     formula = formula
   )
 
+  se_type <- return_list[["se_type"]]
```

Comment:
Anything else need to be set while you are in here? Or is se_type the exception bc it defaults to NULL?

Reply:
It's the exception because of NULL!